tasks/installer-setup.yml

```diff
@@ -0,0 +1,158 @@
+---
+- name: Generate installer trace ID
+  set_fact:
+    datadog_installer_trace_id: "{{ 9999999999999999999 | random }}"
+  when: datadog_installer_enabled
+
+- name: Start duration measurements
+  command: "date +%s%N"
+  register: datadog_installer_start_time
+  changed_when: true
+
+- name: Install datadog-installer package (dnf)
+  dnf:
+    name: "datadog-installer"
+    update_cache: true
+    state: latest # noqa package-latest
+  register: datadog_installer_install_result
+  # Since we want to send telemetry including when the installation failed,
+  # we need to explicitely ignore failures and skip dependent task when one
+  # of its depedency failed
+  # By default ansible will stop on the first error
+  ignore_errors: true
+  when: ansible_facts.os_family in ["RedHat", "Rocky", "AlmaLinux"] and not ansible_check_mode and ansible_pkg_mgr == "dnf"
+
+- name: Install latest datadog-agent package (yum)
+  yum:
+    name: "datadog-installer"
+    update_cache: true
+    state: latest # noqa package-latest
+  register: datadog_installer_install_result
+  ignore_errors: true
+  when: ansible_facts.os_family in ["RedHat", "Rocky", "AlmaLinux"] and not ansible_check_mode and ansible_pkg_mgr == "yum"
```

Comment:
We don't support CentOS 6, should we add a check like we did in the install script like [we did in the install script](https://github.com/DataDog/agent-linux-install-script/pull/212)?

Reply:
done here: https://github.com/DataDog/ansible-datadog/pull/573/files#diff-628b17105eb5471592a85311eb58449633695b482221d729f7e3965cd0ba21cbR14-R17